jobserver/api/releases.py

```diff
@@ -90,8 +90,25 @@ def validate_release_access(request, workspace):
 
 def generate_index(files):
     """Generate a JSON list of files as expected by the SPA."""
+
+    def get_size(rfile):
+        try:
+            return rfile.absolute_path().stat().st_size
+        except FileNotFoundError:  # pragma: no cover
+            return None
+
     return dict(
-        files=[dict(name=k, url=v.get_api_url()) for k, v in files.items()],
+        files=[
+            dict(
+                name=k,
+                url=v.get_api_url(),
+                user=v.created_by.username,
+                date=v.created_at.isoformat(),
+                sha256=v.filehash,
+                size=get_size(v),
+            )
+            for k, v in files.items()
```

Comment:
Now there's a selection of lines using them, could you swap out `k` and `v` for what they represent?  I think they're filename and ReleaseFile? (so maybe `filename` and `f`?)

Reply:
good idea, will do